clients/cobol-lsp-vscode-extension/src/ProfileService.ts

```diff
@@ -63,27 +66,23 @@ export class ProfileService {
         return undefined;
     }
 
-    private async findProfileByDependenciesFile(depFile: vscode.Uri): Promise<string | undefined> {
-        const depName = path.basename(depFile.fsPath, ".dep");
+    private async findProfileByDependenciesFile(programName: string): Promise<string | undefined> {
         for (const doc of vscode.workspace.textDocuments) {
             const docPath = doc.fileName;
             if (!docPath.toLowerCase().endsWith(".cbl")) {
```

Comment:
In fact, we support also `.cob` and `.cobol` case insensitively as valid file extensions for COBOL files.

Reply:
Is it ".cob" (as it now in package.json) or ".cobol"?

Reply:
I think both

Reply:
Are you going to do something with it in this PR or later on?

Reply:
It's there already